src/test/rmc/ForeignItems/missing_fn_fail.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! This tests whether we properly replace missing functions with `assert(false)`
+//! To run this test, do
+//! rmc missing_fn_fail.rs -- lib.c
+
+// rmc-flags: --c-lib src/test/rmc/ForeignItems/lib.c
+
+// https://doc.rust-lang.org/reference/items/external-blocks.html
+// https://doc.rust-lang.org/nomicon/ffi.html
+extern "C" {
```

Comment:
Can we do this without extern "C"? How common is a construct like this in user code?

Reply:
I'd love to do this in pure rust. But I can't easily tell how to generate that case.  Thoughts?